share/completions/kill.fish

```diff
@@ -4,10 +4,21 @@ for i in $__kill_signals
 	set number (echo $i | cut -d " " -f 1)
 	set name (echo $i | cut -d " " -f 2)
 	complete -c kill -o $number -d $name
-	complete -c kill -o $name -d $name
-	complete -c kill -o s -x -a \"$number\tSend\ $name\ signal\"
-	complete -c kill -o s -x -a \"$name\tSend\ $name\ signal\"
+	complete -c kill -o $name
+	complete -c kill -s s -x -a "$number $name"
```

Comment:
That's what it's generating right now.
![screen shot 2015-12-19 at 14 41 53](https://cloud.githubusercontent.com/assets/11729541/11913263/bc50494a-a65e-11e5-976c-555beb93daa1.png)